docs/source/ja/_toctree.yml

```
@@ -29,9 +29,77 @@
     title: LLM を使用した生成
   title: Tutorials
 - sections:
-  - local: generation_strategies
-    title: 生成戦略をカスタマイズする
-  title: Generation
+  # - isExpanded: false
+  #   sections:
+  #   - local: tasks/sequence_classification
+  #     title: Text classification
+  #   - local: tasks/token_classification
+  #     title: Token classification
+  #   - local: tasks/question_answering
+  #     title: Question answering
+  #   - local: tasks/language_modeling
+  #     title: Causal language modeling
+  #   - local: tasks/masked_language_modeling
+  #     title: Masked language modeling
+  #   - local: tasks/translation
+  #     title: Translation
+  #   - local: tasks/summarization
+  #     title: Summarization
+  #   - local: tasks/multiple_choice
+  #     title: Multiple choice
+  #   title: Natural Language Processing
+  # - isExpanded: false
+  #   sections:
+  #   - local: tasks/audio_classification
+  #     title: Audio classification
+  #   - local: tasks/asr
+  #     title: Automatic speech recognition
+  #   title: Audio
+  # - isExpanded: false
+  #   sections:
+  #   - local: tasks/image_classification
+  #     title: Image classification
+  #   - local: tasks/semantic_segmentation
+  #     title: Semantic segmentation
+  #   - local: tasks/video_classification
+  #     title: Video classification
+  #   - local: tasks/object_detection
+  #     title: Object detection
+  #   - local: tasks/zero_shot_object_detection
+  #     title: Zero-shot object detection
+  #   - local: tasks/zero_shot_image_classification
+  #     title: Zero-shot image classification
+  #   - local: tasks/monocular_depth_estimation
+  #     title: Depth estimation
+  #   - local: tasks/image_to_image
+  #     title: Image-to-Image
+  #   - local: tasks/knowledge_distillation_for_image_classification
+  #     title: Knowledge Distillation for Computer Vision
+  #   title: Computer Vision
+  # - isExpanded: false
+  #   sections:
+  #   - local: tasks/image_captioning
+  #     title: Image captioning
+  #   - local: tasks/document_question_answering
+  #     title: Document Question Answering
+  #   - local: tasks/visual_question_answering
+  #     title: Visual Question Answering
+  #   - local: tasks/text-to-speech
+  #     title: Text to speech
+  #   title: Multimodal
+  # - isExpanded: false
+    sections:
+    - local: generation_strategies
+      title: 生成戦略をカスタマイズする
+    title: Generation
+  # - isExpanded: false
+  #   sections:
+  #   - local: tasks/idefics
+  #     title: Image tasks with IDEFICS
+  #   - local: tasks/prompting
+  #     title: LLM prompting guide
+  #   title: Prompting
+  # title: Task Guides
```

This shouldn't be commented out because it needs to close this section in the `toctree`.

```suggestion
    title: Task Guides
```